### PR TITLE
Add base HTML templates

### PR DESCRIPTION
**Pull request by florimondmanca**
# Description

- Create a base HTML template.
- Refactor styles: those that are for the base template, those for each page…
- Use `base.html` throughout the pages.

### CODE REVIEW

**Comment by florimondmanca** [comment]
[Créer un template de base](https://trello.com/c/1YjSGVdk/61-cr%C3%A9er-un-template-de-base)

**Comment by florimondmanca** [comment]
Expect merge conflicts with `master` if #21 gets merged first. :-)

File: series/templates/series/show_details.html [hunk 1]
```diff
@@ -1,66 +1,71 @@
...
 
-<!--Navigation bar-->
-<ul class="navbar">
-    <li class="tab"><a href="{% url 'search_series' %}" id="active">Search</a></li>
-    <li class="tab"><a href="{% url 'followed_series' %}">Favorite shows</a></li>
-    <li class="tab"><a href="{% url 'profile' %}">Profile</a></li>
-    <li class="tab"><a href="{% url 'search_series' %}">About</a></li> <!--TODO link to about page-->
-</ul>
+{% block content %}
+    <h1>{{ show.title }}</h1>
```
Comment (CecileSerene) [inline series/templates/series/show_details.html]: For some reason, it is not centered anymore, even though it is h1. do you override the style somewhere?

File: templates/base.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        <span>your favorite</span> <br/>
+        <span id="show-brand">shows</span>
+    </div>
+</header>
+
+<main>
+    {% block content %}{% endblock %}
+</main>
+
+{% block scripts %}{% endblock %}
```
Comment (CecileSerene) [inline templates/base.html]: Is it possible to easily add a bottom bar, including Legal mentions and Contact? I dont know what we should put there but I feel like every website has one.

File: templates/base.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+    <!-- Extra style sheets -->
+    {% block stylesheets %}{% endblock %}
+</head>
+<body>
+<nav>
+    <ul class="navbar">
+        <li class="tab">
+            <a href="{% url 'search_series' %}" id="active">
+                Search
```
Comment (CecileSerene) [inline templates/base.html]: these texts are not centered anymore either... You can see the underline is longer on the right.

File: series/templates/series/show_details.html (hunk 1 above)
Comment (florimondmanca) [reply series/templates/series/show_details.html]: Fixed!

File: templates/base.html (hunk 2 above)
Comment (florimondmanca) [reply templates/base.html]: Yes, we should. But we have a separate task for that on the board and it's actually a new feature, so I'd argue it is not related to this PR. :-)

File: templates/base.html (hunk 3 above)
Comment (florimondmanca) [reply templates/base.html]: Fixed :tada:

**Pull request merged**